packages/eslint-plugin-lit-a11y/docs/rules/accessible-emoji.md

```diff
@@ -4,6 +4,8 @@
 
 - [Léonie Watson](https://tink.uk/accessible-emoji/)
 
+While many modern user agents are able to announce emoji to screen reader users, this rule is still useful for apps targetting older user agents.
```

Comment:
I think we should at least also add the Deprecated warning in the top heading if we're gonna keep it around, although Im still in favor of just getting rid of it :p

Reply:
added deprecated flag and header text